.github/ISSUE_TEMPLATE/release_checklist.md

```diff
@@ -16,7 +16,7 @@ assignees: ''
 - [ ] Wrap Y ()
 
 **Before release**:
-- [ ] Run `grep 'remove_version="vX.Y.Z"' **/*.py` to check if any deprecations and related tests should be removed in this version
+- [ ] Run `grep --include="*.py" -r 'remove_version="vX.Y.Z"' pygmt` from the base of the repository to check if any deprecations and related tests should be removed in this version
```

Comment:
This command only find the `@deprecate_parameter(..., remove_version="vX.Y.Z")` lines. Do we need to manually find the `test_*_deprecate_*_to_*()` functions?

Reply:
> Or actually, removing the decorator should make the unit tests fail, so we could find the failing tests and remove them

yes, I believe it's easier to remove the failing tests rather than trying to find them manually.